pom.xml

```diff
@@ -36,6 +37,53 @@
         <vitess.vtctld.grpc.port>15999</vitess.vtctld.grpc.port>
     </properties>
 
+    <dependencyManagement>
```

Comment:
The `debezium-parent` module has already `dependencyManagement`, is it a convention to redo the same thing here?

Reply:
@jpechane, good point, I don't think we need those repeated here?

Reply:
@gunnarmorling Explained in https://github.com/debezium/debezium-connector-vitess/pull/3#issuecomment-703583814

Reply:
Ah, I see. Warrants a comment there in the _pom.xml_. I'll add it and merge. Thx!